src/tls_wrap.cc

```diff
@@ -912,6 +912,23 @@ void TLSWrap::EnableSessionCallbacks(
                             wrap);
 }
 
+void TLSWrap::EnableTrace(
+    const FunctionCallbackInfo<Value>& args) {
+  TLSWrap* wrap;
+  ASSIGN_OR_RETURN_UNWRAP(&wrap, args.Holder());
+
+#ifndef OPENSSL_NO_SSL_TRACE
+  if (wrap->ssl_) {
+    BIO* b = BIO_new_fp(stderr,  BIO_NOCLOSE | BIO_FP_TEXT);
+    SSL_set_msg_callback(wrap->ssl_.get(), SSL_trace);
+    SSL_set_msg_callback_arg(wrap->ssl_.get(), b);
+
+    args.GetReturnValue().Set(true);
+  } else {
+    args.GetReturnValue().Set(false);
```

Comment:
The returned value isn’t used anywhere?

Reply:
It was intended to be returned, I'll fix that and document it.

Reply:
I removed the return value, I don't think it has enough value to bother documenting.